internal_docs/releasing.md

```diff
@@ -29,21 +29,26 @@ v8.0.0 // After implementing backwards incompatible changes
 
 3. Checkout a new branch called release-[new version number].
 
-4. Update the version number in [VERSION.txt](/VERSION.txt) and update "version" in [package.json](/package.json#L4).
+4. Update the [CHANGELOG.md](/CHANGELOG.md) by:
 
-5. Run `npm install` to update `package-lock.json`.
+  - changing the 'Unreleased' heading to the new version-number and release-type - for example, '12.0.1 (Fix release)'
```

Comment:
really tiny thing but is there a reason to hyphenate 'version number'? we don't elsewhere

Reply:
We hyphenated it in the corresponding step of the [Frontend release doc](https://github.com/alphagov/govuk-frontend/blob/main/docs/releasing/publishing.md#:~:text=Update%20the%20CHANGELOG.md%20by%3A) (which is where I copied this content from, ha). But happy to change both if that goes against practice elsewhere!

Reply:
should be your call, I dont really understand the need to personally

Reply:
One argument _for_ the hyphen: it makes it clearer that 'new' refers to the 'version number' and not the 'number'.

Reply:
very happy to take your lead and more importantly @NoraGDS has given a thumbs up